src/Upgrade/Infrastructure/VersionControlSystem/SourceCodeProvider/GitHub/GitHubSourceCodeProvider.php

```diff
@@ -149,7 +164,35 @@ public function buildBlockerTextBlock(ValidatorViolationDto $blocker): string
             '> [!IMPORTANT] %s> <b>%s.</b> %s',
             PHP_EOL,
             $blocker->getTitle(),
-            $blocker->getMessage(),
+            $this->buildMessageWithTruncatedTrace($blocker->getMessage()),
         ) . PHP_EOL;
     }
+
+    /**
+     * @param string $message
+     *
+     * @return string
+     */
+    public function buildMessageWithTruncatedTrace(string $message): string
```

Comment:
```suggestion
    protected function buildMessageWithTruncatedTrace(string $message): string
```

Reply:
This method is used in tests, e.g. GitHubSourceCodeProviderTest::testBuildBlockerTextBlockTruncatesErrorTrace, so it has to be public

Reply:
We do not make smth public just for tests, please, fix it